### ISSUE TITLE
Can aj work together with tokio?

### ISSUE THREAD

**Issue by Liangdi** [cptrodgers/aj#9]
i wish can use aj in existing apps with tokio

**Comment by cptrodgers** [comment]
Hey @Liangdi thanks for question, 

technically aj can work with tokio engine and ecosystem because actix is based on tokio. 

So, you can use it with tokio. Just start aj as normal, we will start a thread to handle actix runtime if not found actix rt. 

If there are any issues with it, post here then I can help you to figure out it.

**Comment by Liangdi** [comment]
```rust

use aj::{job, AJ};
use tokio::time::sleep;

#[job]
async fn hello(name: String) {
    println!("Hello {name}");
}

#[tokio::main]
async fn main() {

    AJ::quick_start();
    let _r = hello::run("Tokio".to_string()).await;

    sleep(std::time::Duration::from_secs(10)).await;
}

```

this code does not work, error info is
```
Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.
```

use `#[aj::main]`  works well

**Comment by cptrodgers** [comment]
Got it, I'm debugging it, will update an patch to fix it soon @Liangdi .

**Comment by cptrodgers** [comment]
My mistake, I forgot to spawn new thread to handle `actix_rt` runtime then it conflict with current tokio rt.

**Comment by cptrodgers** [comment]
Hey @Liangdi ,  I just published v0.6.6 to fixed conflict runtime issue. Check out the example with tokio here: https://github.com/cptrodgers/aj/blob/master/examples/tokio/src/main.rs

Let's me know if there are any issue with new version 🙏

**Comment by Liangdi** [comment]
> Hey @Liangdi , I just published v0.6.6 to fixed conflict runtime issue. Check out the example with tokio here: https://github.com/cptrodgers/aj/blob/master/examples/tokio/src/main.rs
> 
> Let's me know if there are any issue with new version 🙏

great works, it' ok now!

**Comment by cptrodgers** [comment]
Awesome, I will close the issue. Thanks for the feedback @Liangdi
